caption/stories/index.stories.js

```diff
@@ -0,0 +1,23 @@
+import '~/caption/dockit-caption.define';
+
+export default {
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+export const caption_for_image = () => /*html*/ `
+  <div>
```

Comment:
why extra div wrapper?

Reply:
Was needed because of the story layout that components displays with flex row. Could have handled it via story layout but it's not necessary any more since I dropped this component altogether and went for `figure` and `figcaption` with a bit of styling in our doc-layout as you and Georges suggested!